blazor/skeleton/getting-started.md

```diff
@@ -282,6 +282,47 @@ N> Syncfusion recommends to reference scripts using [Static Web Assets](https://
 
 ![Blazor Skeleton Component](./images/blazor-skeleton-component.png)
 
+## Initialize as CSS component
+
+The Blazor Skeleton component can be initialized as a CSS component by utilizing predefined class names, and then you can apply CSS styles to control the width and height of the skeleton shape.
+
+| Class | Description |
+| -------- | -------- |
+| e-skeleton | Defines the Skeleton class. |
+| e-skeleton-text |  Defines the Skeleton shape as text. |
+| e-shimmer-pulse |  Defines the animation as pulse effect. |
+
+{% tabs %}
+{% highlight razor %}
+
+@using Syncfusion.Blazor.Notifications
+
+<div class="content e-skeleton e-skeleton-text e-shimmer-pulse"></div>
+
+<style>
+    .content {
+        width:50%;
+        height:20px;
+    }
+</style>
+
+{% endhighlight %}
+{% endtabs %}
+
+![Blazor Skeleton Css component](./images/blazor-skeleton-css-component.png)
+
+## Visible
+
+You can use the [Visible](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Notifications.SfSkeleton.html#Syncfusion_Blazor_Notifications_SfSkeleton_Visible) property which defines the visible state of Skeleton.
+
+{% tabs %}
+{% highlight razor %}
+
+<SfSkeleton Shape=SkeletonType.Circle Width="60px" Visible=false></SfSkeleton>
+
+{% endhighlight %}
+{% endtabs %}
+
 ## Skeleton Shapes
```

Comment:
Remove this topic here, as it is added as a separate page

Reply:
Removed the topic